wix/InstallerChangelog.md

```diff
@@ -8,6 +8,11 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## 2024-05-02
+### Changed
+- Update to wix v5
+- Fixed windows Arm installation path
+
```

Comment:
```suggestion
- Update IcedTea-Web to 1.8.8

```

Reply:
I'd prefer to not call this out in the changelog. It's unrelated to the changes (only affect GitHub CI)